operations/node_management/Rebuild_NCNs/Identify_Nodes_and_Update_Metadata.md

```diff
@@ -59,4 +59,4 @@ This section applies to all node types. The commands in this section assume the
 
 ## Next Step
 
-Proceed to the next step to [Power Cycle and Rebuild Nodes](Power_Cycle_and_Rebuild_Nodes.md). Otherwise, return to the main [Rebuild NCNs](Rebuild_NCNs.md) page.
+Return to the main [Rebuild NCNs](Rebuild_NCNs.md#storage_node) page and proceed with the next step.
```

Comment:
I wonder if it would be more clear to say 
"If doing this part of a NCN rebuild, return to the main page and proceed with the next  step in the NCN rebuild." This is just an opinion because maybe the user won't think they are done with this since there is another step?

Reply:
yeah -- this is intentional -- we've changed all the 'next' links to go back to the main page, and this 'flow' is really only for storage nodes..

Reply:
after more discussion, rephrased all the next links..